### PR TITLE
Adding empty state pattern

### PR DESCRIPTION
**Pull request by cshinn**
Adds the empty state pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
yeap, this one also looks good, thanks @cshinn 🚀

File: pattern-library/communication/empty-state/design/design.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Empty State
+
+## Description
+
+![Empty State with Callouts](img/empty-state.png)
+
+1. **Icon** (optional)**:** If an object icon is associated with this view, it can be displayed here.
+1. **Empty State Title:** Give it a name.
+1. **Text** (optional)**:** This text should be personal and helpful. It should minimize the user’s effort to complete tasks.
+1. **Primary Action Button:** The primary action is displayed as a prominent blue button. See the Action Labels section for more information about terminology and wording specific to action labels.
```
Comment (LHinson) [inline pattern-library/communication/empty-state/design/design.md]: On the site, the word "Action Labels" links to http://www.patternfly.org/styles/terminology-and-wording/#action-labels

**Comment by andresgalante** [comment]
@LHinson ready to merge! it has the example image.

@cshinn thanks!! you are the 💣

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**